_posts/2016-03-21-windows-support.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Windows support update
+categories: tech-issues future-plans
```

Comment:
We don't have anything like it yet but would "deployment" be better than "tech-issues"?